### PR TITLE
luminous: qa: move mds/client config to qa from teuthology ceph.conf.template

### PR DESCRIPTION
**Pull request by smithfarm**
http://tracker.ceph.com/issues/24842
http://tracker.ceph.com/issues/26904

### CODE REVIEW

**Comment by smithfarm** [comment]
@batrick Ready for review

**Comment by yuriw** [comment]
[wip-yuri2-testing-2018-09-13-1634-luminous](https://trello.com/c/kzbWeNmq/403-wip-yuri2-testing-2018-09-13-1634-luminous)

**Comment by batrick** [comment]
needs https://github.com/ceph/ceph/pull/24134

File: qa/suites/fs/snaps/conf [hunk 1]
```diff
@@ -0,0 +1 @@
+.qa/cephfs/conf
```
Comment (batrick) [inline qa/suites/fs/snaps/conf]: This suite does not exist in luminous.

**Comment by batrick** [comment]
@smithfarm after fixing this, please ensure `teuthology-suite ... --dry-run` can schedule this PR along with #24134

**Comment by batrick** [comment]
````
(venv) pdonnell@teuthology ~$ teuthology-suite -v --ceph-repo https://github.com/ceph/ceph-ci.git --suite-repo https://github.com/batrick/ceph.git -c wip-yuri2-testing-2018-09-13-1634-luminous -m smithi -s fs --subset 1/10 -k distro -p 60 --suite-branch wip-pdonnell-testing-20180917.180254 --dry-run
2018-09-17 18:04:22,898.898 INFO:teuthology.suite:Using random seed=1964
2018-09-17 18:04:22,899.899 INFO:teuthology.suite.run:kernel sha1: distro
2018-09-17 18:04:23,144.144 DEBUG:teuthology.repo_utils:git ls-remote https://github.com/ceph/ceph-ci wip-yuri2-testing-2018-09-13-1634-luminous -> 90d2ea2fa9fbe252d57f9ae24813e0caf482579f
2018-09-17 18:04:23,144.144 INFO:teuthology.suite.run:ceph sha1: 90d2ea2fa9fbe252d57f9ae24813e0caf482579f
2018-09-17 18:04:23,144.144 DEBUG:teuthology.suite.util:Defaults for machine_type smithi distro centos: arch=x86_64, release=centos/7, pkg_type=rpm
2018-09-17 18:04:23,145.145 DEBUG:teuthology.packaging:Querying https://shaman.ceph.com/api/search?status=ready&project=ceph&flavor=default&distros=centos%2F7%2Fx86_64&sha1=90d2ea2fa9fbe252d57f9ae24813e0caf482579f
2018-09-17 18:04:23,443.443 INFO:teuthology.suite.run:ceph version: 12.2.8-93.g90d2ea2
2018-09-17 18:04:23,443.443 INFO:teuthology.suite.run:teuthology branch: master
2018-09-17 18:04:23,670.670 DEBUG:teuthology.repo_utils:git ls-remote https://github.com/batrick/ceph.git wip-pdonnell-testing-20180917.180254 -> d2ae49c6210f16f1c9b1594f94c2cfd5c547ae5a
2018-09-17 18:04:23,926.926 DEBUG:teuthology.repo_utils:git ls-remote https://github.com/batrick/ceph.git wip-pdonnell-testing-20180917.180254 -> d2ae49c6210f16f1c9b1594f94c2cfd5c547ae5a
2018-09-17 18:04:23,926.926 INFO:teuthology.suite.run:ceph branch: wip-pdonnell-testing-20180917.180254 d2ae49c6210f16f1c9b1594f94c2cfd5c547ae5a
2018-09-17 18:04:23,928.928 INFO:teuthology.repo_utils:Cloning https://github.com/batrick/ceph.git wip-pdonnell-testing-20180917.180254 from upstream
2018-09-17 18:04:28,678.678 INFO:teuthology.repo_utils:Resetting repo at /home/pdonnell/src/github.com_batrick_ceph_wip-pdonnell-testing-20180917.180254 to branch wip-pdonnell-testing-20180917.180254
2018-09-17 18:04:29,134.134 DEBUG:teuthology.suite.run:Suite fs in /home/pdonnell/src/github.com_batrick_ceph_wip-pdonnell-testing-20180917.180254/qa/suites/fs
2018-09-17 18:04:29,134.134 INFO:teuthology.suite.build_matrix:Subset=1/10
Traceback (most recent call last):
  File "/home/pdonnell/teuthology/venv/bin/teuthology-suite", line 11, in <module>
    load_entry_point('teuthology', 'console_scripts', 'teuthology-suite')()
  File "/home/pdonnell/teuthology/scripts/suite.py", line 143, in main
    return teuthology.suite.main(args)
  File "/home/pdonnell/teuthology/teuthology/suite/__init__.py", line 93, in main
    run.prepare_and_schedule()
  File "/home/pdonnell/teuthology/teuthology/suite/run.py", line 341, in prepare_and_schedule
    num_jobs = self.schedule_suite()
  File "/home/pdonnell/teuthology/teuthology/suite/run.py", line 492, in schedule_suite
    build_matrix(suite_path, subset=self.args.subset, seed=self.args.seed)
  File "/home/pdonnell/teuthology/teuthology/suite/build_matrix.py", line 57, in build_matrix
    mat, first, matlimit = _get_matrix(path, subset)
  File "/home/pdonnell/teuthology/teuthology/suite/build_matrix.py", line 67, in _get_matrix
    mat = _build_matrix(path, mincyclicity=outof)
  File "/home/pdonnell/teuthology/teuthology/suite/build_matrix.py", line 146, in _build_matrix
    fn)
  File "/home/pdonnell/teuthology/teuthology/suite/build_matrix.py", line 146, in _build_matrix
    fn)
  File "/home/pdonnell/teuthology/teuthology/suite/build_matrix.py", line 146, in _build_matrix
    fn)
  File "/home/pdonnell/teuthology/teuthology/suite/build_matrix.py", line 84, in _build_matrix
    raise IOError('%s does not exist (abs %s)' % (path, os.path.abspath(path)))
IOError: /home/pdonnell/src/github.com_batrick_ceph_wip-pdonnell-testing-20180917.180254/qa/suites/fs/upgrade/snaps/conf does not exist (abs /home/pdonnell/src/github.com_batrick_ceph_wip-pdonnell-testing-20180917.180254/qa/suites/fs/upgrade/snaps/conf)
````

For posterity

**Comment by yuriw** [comment]
@smithfarm pls add tags back when fixed.

**Comment by smithfarm** [comment]
@batrick: Looks like it's fixed:

```
(virtualenv) smithfarm@teuthology:~$ teuthology-suite --machine-type smithi --email ncutler@suse.cz --priority 101 --ceph-repo https://github.com/ceph/ceph.git --ceph luminous --newest 10 --suite-repo https://github.com/smithfarm/ceph.git --suite-branch wip-24842-luminous --suite fs/snaps --dry-run      
2018-09-27 14:03:36,835.835 INFO:teuthology.suite:Using random seed=2436
2018-09-27 14:03:36,836.836 INFO:teuthology.suite.run:kernel sha1: distro
2018-09-27 14:03:37,181.181 INFO:teuthology.suite.run:ceph sha1: 1fdbedfc6a08ac9363ac6b3a3390cda1f877ed9e
2018-09-27 14:03:37,181.181 INFO:teuthology.suite.run:skipping ceph package verification
2018-09-27 14:03:37,181.181 INFO:teuthology.suite.run:teuthology branch: master
2018-09-27 14:03:37,793.793 INFO:teuthology.suite.run:ceph branch: wip-24842-luminous 8a5e0f5aaf1e84730e9ad712cb7ee48882d72b9b
2018-09-27 14:03:37,805.805 INFO:teuthology.repo_utils:Fetching wip-24842-luminous from upstream
2018-09-27 14:03:38,232.232 INFO:teuthology.repo_utils:Resetting repo at /home/smithfarm/src/github.com_smithfarm_ceph_wip-24842-luminous to branch wip-24842-luminous
2018-09-27 14:03:38,667.667 INFO:teuthology.suite.run:Suite fs:snaps in /home/smithfarm/src/github.com_smithfarm_ceph_wip-24842-luminous/qa/suites/fs/snaps generated 5 jobs (not yet filtered)
...
... <ABRIDGED FOR BREVITY>
...
2018-09-27 14:03:38,955.955 INFO:teuthology.suite.run:Test results viewable at http://pulpito.front.sepia.ceph.com:80/smithfarm-2018-09-27_14:03:36-fs:snaps-luminous-distro-basic-smithi/
```

File: qa/suites/fs/snaps/conf (hunk 1 above)
Comment (smithfarm) [reply qa/suites/fs/snaps/conf]: @batrick Not sure what you mean here. There does seem to be an `fs/snaps` suite in luminous: https://github.com/ceph/ceph/tree/luminous/qa/suites/fs/snaps

**Comment by yuriw** [comment]
[wip-yuri5-testing-2018-09-27-1658-luminous](https://trello.com/c/uhzsgxSi/416-wip-yuri5-testing-2018-09-27-1658-luminous)

**Comment by yuriw** [comment]
I think its not fixed, but will validate and test this PR by itself

**Comment by yuriw** [comment]
[wip-yuri2-testing-2018-09-27-2005-luminous](https://trello.com/c/Q4W6VWhu/417-wip-yuri2-testing-2018-09-27-2005-luminous)

**Comment by yuriw** [comment]
Still broken @smithfarm :

teuthology-suite -v --ceph-repo https://github.com/ceph/ceph-ci.git --suite-repo https://github.com/ceph/ceph-ci.git -c $CEPH_BRANCH -m $MACHINE_NAME -s fs --subset 1/10 -k distro -p 60 -e $CEPH_QA_MAIL --suite-branch $CEPH_BRANCH; teuthology-suite -v --ceph-repo https://github.com/ceph/ceph-ci.git --suite-repo https://github.com/ceph/ceph-ci.git -c $CEPH_BRANCH -m $MACHINE_NAME -s kcephfs --subset 1/10 -k testing -p 60 -e $CEPH_QA_MAIL --suite-branch $CEPH_BRANCH;
2018-09-27 21:50:58,518.518 INFO:teuthology.suite:Using random seed=2140
2018-09-27 21:50:58,520.520 INFO:teuthology.suite.run:kernel sha1: distro
2018-09-27 21:50:58,764.764 DEBUG:teuthology.repo_utils:git ls-remote https://github.com/ceph/ceph-ci wip-yuri2-testing-2018-09-27-2005-luminous -> 312a4a925a4b3ed3efc7b7ec5349a66da442f501
2018-09-27 21:50:58,764.764 INFO:teuthology.suite.run:ceph sha1: 312a4a925a4b3ed3efc7b7ec5349a66da442f501
2018-09-27 21:50:58,765.765 DEBUG:teuthology.suite.util:Defaults for machine_type smithi distro centos: arch=x86_64, release=centos/7, pkg_type=rpm
2018-09-27 21:50:58,765.765 DEBUG:teuthology.packaging:Querying https://shaman.ceph.com/api/search?status=ready&project=ceph&flavor=default&distros=centos%2F7%2Fx86_64&sha1=312a4a925a4b3ed3efc7b7ec5349a66da442f501
2018-09-27 21:50:59,059.059 INFO:teuthology.suite.run:ceph version: 12.2.8-269.g312a4a9
2018-09-27 21:50:59,059.059 INFO:teuthology.suite.run:teuthology branch: master
2018-09-27 21:50:59,304.304 DEBUG:teuthology.repo_utils:git ls-remote https://github.com/ceph/ceph-ci.git wip-yuri2-testing-2018-09-27-2005-luminous -> 312a4a925a4b3ed3efc7b7ec5349a66da442f501
2018-09-27 21:50:59,539.539 DEBUG:teuthology.repo_utils:git ls-remote https://github.com/ceph/ceph-ci.git wip-yuri2-testing-2018-09-27-2005-luminous -> 312a4a925a4b3ed3efc7b7ec5349a66da442f501
2018-09-27 21:50:59,540.540 INFO:teuthology.suite.run:ceph-ci branch: wip-yuri2-testing-2018-09-27-2005-luminous 312a4a925a4b3ed3efc7b7ec5349a66da442f501
2018-09-27 21:50:59,547.547 INFO:teuthology.repo_utils:Cloning https://github.com/ceph/ceph-ci.git wip-yuri2-testing-2018-09-27-2005-luminous from upstream
2018-09-27 21:51:03,413.413 INFO:teuthology.repo_utils:Resetting repo at /home/yuriw/src/github.com_ceph_ceph-c_wip-yuri2-testing-2018-09-27-2005-luminous to branch wip-yuri2-testing-2018-09-27-2005-luminous
2018-09-27 21:51:03,776.776 DEBUG:teuthology.suite.run:Suite fs in /home/yuriw/src/github.com_ceph_ceph-c_wip-yuri2-testing-2018-09-27-2005-luminous/qa/suites/fs
2018-09-27 21:51:03,776.776 INFO:teuthology.suite.build_matrix:Subset=1/10
Traceback (most recent call last):
  File "/home/yuriw/teuthology/virtualenv/bin/teuthology-suite", line 11, in <module>
    load_entry_point('teuthology', 'console_scripts', 'teuthology-suite')()
  File "/home/yuriw/teuthology/scripts/suite.py", line 143, in main
    return teuthology.suite.main(args)
  File "/home/yuriw/teuthology/teuthology/suite/__init__.py", line 120, in main
    run.prepare_and_schedule()
  File "/home/yuriw/teuthology/teuthology/suite/run.py", line 341, in prepare_and_schedule
    num_jobs = self.schedule_suite()
  File "/home/yuriw/teuthology/teuthology/suite/run.py", line 492, in schedule_suite
    build_matrix(suite_path, subset=self.args.subset, seed=self.args.seed)
  File "/home/yuriw/teuthology/teuthology/suite/build_matrix.py", line 57, in build_matrix
    mat, first, matlimit = _get_matrix(path, subset)
  File "/home/yuriw/teuthology/teuthology/suite/build_matrix.py", line 67, in _get_matrix
    mat = _build_matrix(path, mincyclicity=outof)
  File "/home/yuriw/teuthology/teuthology/suite/build_matrix.py", line 146, in _build_matrix
    fn)
  File "/home/yuriw/teuthology/teuthology/suite/build_matrix.py", line 146, in _build_matrix
    fn)
  File "/home/yuriw/teuthology/teuthology/suite/build_matrix.py", line 146, in _build_matrix
    fn)
  File "/home/yuriw/teuthology/teuthology/suite/build_matrix.py", line 84, in _build_matrix
    raise IOError('%s does not exist (abs %s)' % (path, os.path.abspath(path)))
IOError: /home/yuriw/src/github.com_ceph_ceph-c_wip-yuri2-testing-2018-09-27-2005-luminous/qa/suites/fs/upgrade/snaps/conf does not exist (abs /home/yuriw/src/github.com_ceph_ceph-c_wip-yuri2-testing-2018-09-27-2005-luminous/qa/suites/fs/upgrade/snaps/conf)
2018-09-27 21:51:04,565.565 INFO:teuthology.suite:Using random seed=467
2018-09-27 21:51:04,566.566 DEBUG:teuthology.suite.util:Defaults for machine_type smithi distro centos: arch=x86_64, release=centos/7, pkg_type=rpm
2018-09-27 21:51:04,567.567 DEBUG:teuthology.packaging:Querying https://shaman.ceph.com/api/search?status=ready&project=kernel&flavor=default&distros=centos%2F7%2Fx86_64&ref=testing
2018-09-27 21:51:04,967.967 INFO:teuthology.suite.run:kernel sha1: 14cc4f55d2aa69ee1e1cf81a874bcf2cbf0eb9b8
2018-09-27 21:51:05,373.373 DEBUG:teuthology.repo_utils:git ls-remote https://github.com/ceph/ceph-ci wip-yuri2-testing-2018-09-27-2005-luminous -> 312a4a925a4b3ed3efc7b7ec5349a66da442f501
2018-09-27 21:51:05,373.373 INFO:teuthology.suite.run:ceph sha1: 312a4a925a4b3ed3efc7b7ec5349a66da442f501
2018-09-27 21:51:05,374.374 DEBUG:teuthology.suite.util:Defaults for machine_type smithi distro centos: arch=x86_64, release=centos/7, pkg_type=rpm
2018-09-27 21:51:05,374.374 DEBUG:teuthology.packaging:Querying https://shaman.ceph.com/api/search?status=ready&project=ceph&flavor=default&distros=centos%2F7%2Fx86_64&sha1=312a4a925a4b3ed3efc7b7ec5349a66da442f501
2018-09-27 21:51:05,591.591 INFO:teuthology.suite.run:ceph version: 12.2.8-269.g312a4a9
2018-09-27 21:51:05,591.591 INFO:teuthology.suite.run:teuthology branch: master
2018-09-27 21:51:05,918.918 DEBUG:teuthology.repo_utils:git ls-remote https://github.com/ceph/ceph-ci.git wip-yuri2-testing-2018-09-27-2005-luminous -> 312a4a925a4b3ed3efc7b7ec5349a66da442f501
2018-09-27 21:51:06,184.184 DEBUG:teuthology.repo_utils:git ls-remote https://github.com/ceph/ceph-ci.git wip-yuri2-testing-2018-09-27-2005-luminous -> 312a4a925a4b3ed3efc7b7ec5349a66da442f501
2018-09-27 21:51:06,184.184 INFO:teuthology.suite.run:ceph-ci branch: wip-yuri2-testing-2018-09-27-2005-luminous 312a4a925a4b3ed3efc7b7ec5349a66da442f501
2018-09-27 21:51:06,186.186 DEBUG:teuthology.repo_utils:Setting repo remote to https://github.com/ceph/ceph-ci.git
2018-09-27 21:51:06,193.193 INFO:teuthology.repo_utils:Fetching wip-yuri2-testing-2018-09-27-2005-luminous from upstream
2018-09-27 21:51:06,613.613 INFO:teuthology.repo_utils:Resetting repo at /home/yuriw/src/github.com_ceph_ceph-c_wip-yuri2-testing-2018-09-27-2005-luminous to branch wip-yuri2-testing-2018-09-27-2005-luminous
2018-09-27 21:51:06,978.978 DEBUG:teuthology.suite.run:Suite kcephfs in /home/yuriw/src/github.com_ceph_ceph-c_wip-yuri2-testing-2018-09-27-2005-luminous/qa/suites/kcephfs
2018-09-27 21:51:06,978.978 INFO:teuthology.suite.build_matrix:Subset=1/10
Traceback (most recent call last):
  File "/home/yuriw/teuthology/virtualenv/bin/teuthology-suite", line 11, in <module>
    load_entry_point('teuthology', 'console_scripts', 'teuthology-suite')()
  File "/home/yuriw/teuthology/scripts/suite.py", line 143, in main
    return teuthology.suite.main(args)
  File "/home/yuriw/teuthology/teuthology/suite/__init__.py", line 120, in main
    run.prepare_and_schedule()
  File "/home/yuriw/teuthology/teuthology/suite/run.py", line 341, in prepare_and_schedule
    num_jobs = self.schedule_suite()
  File "/home/yuriw/teuthology/teuthology/suite/run.py", line 492, in schedule_suite
    build_matrix(suite_path, subset=self.args.subset, seed=self.args.seed)
  File "/home/yuriw/teuthology/teuthology/suite/build_matrix.py", line 57, in build_matrix
    mat, first, matlimit = _get_matrix(path, subset)
  File "/home/yuriw/teuthology/teuthology/suite/build_matrix.py", line 67, in _get_matrix
    mat = _build_matrix(path, mincyclicity=outof)
  File "/home/yuriw/teuthology/teuthology/suite/build_matrix.py", line 146, in _build_matrix
    fn)
  File "/home/yuriw/teuthology/teuthology/suite/build_matrix.py", line 130, in _build_matrix
    item=fn)
  File "/home/yuriw/teuthology/teuthology/suite/build_matrix.py", line 155, in _build_matrix
    return matrix.Sum(item, submats)
  File "/home/yuriw/teuthology/teuthology/suite/matrix.py", line 250, in __init__
    "Sum requires non-empty _submats"
AssertionError: Sum requires non-empty _submats

File: qa/suites/fs/snaps/conf (hunk 1 above)
Comment (batrick) [reply qa/suites/fs/snaps/conf]: Sorry I thought it was `fs/upgrade`, nevermind.

File: qa/suites/fs/upgrade/snaps/conf [hunk 2]
```diff
@@ -0,0 +1 @@
+.qa/cephfs/conf
```
Comment (batrick) [inline qa/suites/fs/upgrade/snaps/conf]: @smithfarm this is the one I was meaning to say does not exist.

**Comment by smithfarm** [comment]
2018-09-28 06:27:17,176.176 INFO:teuthology.suite.run:Suite fs in /home/smithfarm/src/github.com_smithfarm_ceph_wip-24842-luminous/qa/suites/fs scheduled 516 jobs.

**Comment by yuriw** [comment]
[wip-yuri3-testing-2018-09-28-2208-luminous](https://trello.com/c/VJzztDob/423-wip-yuri3-testing-2018-09-28-2208-luminous)

**Comment by yuriw** [comment]
@smithfarm see `kcephfs`

CEPH_QA_MAIL="ceph-qa@ceph.com"; MACHINE_NAME=smithi; CEPH_BRANCH=wip-yuri3-testing-2018-09-28-2208-luminous

teuthology-suite -v --ceph-repo https://github.com/ceph/ceph-ci.git --suite-repo https://github.com/ceph/ceph-ci.git -c $CEPH_BRANCH -m $MACHINE_NAME -s kcephfs --subset 1/10 -k testing -p 60 -e $CEPH_QA_MAIL --suite-branch $CEPH_BRANCH

........
2018-09-29 15:12:54,109.109 INFO:teuthology.suite.run:ceph-ci branch: wip-yuri3-testing-2018-09-28-2208-luminous 0bd823c8742653abb3b37c3ea51f4f40ce17ad9a
2018-09-29 15:12:54,111.111 DEBUG:teuthology.repo_utils:Setting repo remote to https://github.com/ceph/ceph-ci.git
2018-09-29 15:12:54,120.120 INFO:teuthology.repo_utils:Fetching wip-yuri3-testing-2018-09-28-2208-luminous from upstream
2018-09-29 15:12:54,510.510 INFO:teuthology.repo_utils:Resetting repo at /home/yuriw/src/github.com_ceph_ceph-c_wip-yuri3-testing-2018-09-28-2208-luminous to branch wip-yuri3-testing-2018-09-28-2208-luminous
2018-09-29 15:12:54,597.597 DEBUG:teuthology.suite.run:Suite kcephfs in /home/yuriw/src/github.com_ceph_ceph-c_wip-yuri3-testing-2018-09-28-2208-luminous/qa/suites/kcephfs
2018-09-29 15:12:54,597.597 INFO:teuthology.suite.build_matrix:Subset=1/10
Traceback (most recent call last):
  File "/home/yuriw/teuthology/virtualenv/bin/teuthology-suite", line 11, in <module>
    load_entry_point('teuthology', 'console_scripts', 'teuthology-suite')()
  File "/home/yuriw/teuthology/scripts/suite.py", line 143, in main
    return teuthology.suite.main(args)
  File "/home/yuriw/teuthology/teuthology/suite/__init__.py", line 120, in main
    run.prepare_and_schedule()
  File "/home/yuriw/teuthology/teuthology/suite/run.py", line 341, in prepare_and_schedule
    num_jobs = self.schedule_suite()
  File "/home/yuriw/teuthology/teuthology/suite/run.py", line 492, in schedule_suite
    build_matrix(suite_path, subset=self.args.subset, seed=self.args.seed)
  File "/home/yuriw/teuthology/teuthology/suite/build_matrix.py", line 57, in build_matrix
    mat, first, matlimit = _get_matrix(path, subset)
  File "/home/yuriw/teuthology/teuthology/suite/build_matrix.py", line 67, in _get_matrix
    mat = _build_matrix(path, mincyclicity=outof)
  File "/home/yuriw/teuthology/teuthology/suite/build_matrix.py", line 146, in _build_matrix
    fn)
  File "/home/yuriw/teuthology/teuthology/suite/build_matrix.py", line 130, in _build_matrix
    item=fn)
  File "/home/yuriw/teuthology/teuthology/suite/build_matrix.py", line 155, in _build_matrix
    return matrix.Sum(item, submats)
  File "/home/yuriw/teuthology/teuthology/suite/matrix.py", line 250, in __init__
    "Sum requires non-empty _submats"
AssertionError: Sum requires non-empty _submats

File: qa/suites/kcephfs/recovery/debug/mds_client.yaml [hunk 3]
```diff
@@ -1,12 +0,0 @@
-overrides:
```
Comment (batrick) [inline qa/suites/kcephfs/recovery/debug/mds_client.yaml]: The containing directory 'qa/suites/kcephfs/recovery/debug` needs deleted too. Ditto for `recovery/dirfrag`.

**Comment by smithfarm** [comment]
@yuriw Please try again?

**Comment by yuriw** [comment]
[wip-yuri4-testing-2018-10-02-1655-luminous](https://trello.com/c/7L0AI1WJ/428-wip-yuri4-testing-2018-10-02-1655-luminous)

**Comment by yuriw** [comment]
@smithfarm see more promise, trying ...

**Comment by yuriw** [comment]
@smithfarm @batrick merging but we need to reduce number of nodes required to run these tests

http://pulpito.ceph.com/yuriw-2018-10-03_16:41:08-kcephfs-wip-yuri4-testing-2018-10-02-1655-luminous-testing-basic-smithi/

otherwise they will be timing out trying to lock nodes.

**Pull request merged**